data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,495 @@ mission "Unfettered returning home"
+			label lucky??
+			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing. You feel like you hear a robotic voice cry out, but it's hard to tell with all the noise.`
+			`	Unfortunately, the crash is right on top of you, and it seems more like a landing than an actual crash, especially considering the Hai are left mostly untouched, having taken a few leaps back right in time.`
+				die
+			label lucky?
+			`	Probably thanks to the fact the ship's shields are down, and its depreciated state, you manage to land a very lucky shot, going through a layer of rust and severely damaging the reactor of the ship. The ship's speakers cry out, "MY REACTOR... NOT MY REACTOR!"`
+			`	It might have seemed lucky, that is, until it blows up, taking you and everyone else on the beach down with it.`
+				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react, probably not viewing you as a threat. You still have time to do something else or try again. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>, with it as your flagship and preferably without any other ships."
+	clearance
+	# deadline to make sure you don't just enjoy free rep
+	deadline 30
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`Upon returning to the spaceport, one could say the cargo finds you more than you find it. A worker immediately greets you while others begin to fill your ship with crates, which a quick check shows have already been allocated for this ship in your name. All that is left to do is to bring it to <destination> by <date> for the promised sum of <payment>.`
+			choice
+				`	(Prepare for takeoff.)`
+			`	As you head to the bridge, preparing to take off from the planet, you notice a message on the command panel, from your "associates," probably:`
+			`	"Use the Anomalocaris as your flagship.`
+			`	"Do not bring any other ships, as they are not needed for the cargo and would only be a liability at best in Unfettered space, when they could easily provoke them at worst."`
+			`	Given the potential dangers of that area of space, it would be a good idea to familiarize yourself with the design of the Anomalocaris and even customize it more to your own liking - especially if you plan to deliver the cargo in only this single ship, as requested.`
```

Comment:
```suggestion
			`	Given the potential dangers of that area of space, it would be a good idea to familiarize yourself with the design of the Anomalocaris, or perhaps customize it more to your own liking, especially if you plan to deliver the cargo in only this single ship, as requested.`
```